package.json

```diff
@@ -111,7 +112,8 @@
     "sqlite3": "^5.0.0",
     "ts-node": "^9.0.0",
     "typeorm-aurora-data-api-driver": "^1.4.0",
-    "typescript": "~3.6.0"
+    "typescript": "~3.6.0",
+    "uuid": "^8.3.1"
```

Comment:
Should use the internal uuid generation:

https://github.com/typeorm/typeorm/blob/c4a36da62593469436b074873eba186f0f8b990d/src/util/RandomGenerator.ts#L149